docs/advanced.rst

```diff
@@ -0,0 +1,605 @@
+Advanced Usage
+==============
```

Comment:
What's this addition? Looks like you had an issue rebasing

Reply:
@rossmechanic  - sorry about that. Have fixed.